index.html

```diff
@@ -369,24 +370,80 @@ <h1>Social Media Monitoring</h1>
   <div class="content">
   	<div class="row">
     <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
-    <p>Currently used by <a href="http://directives.chicagopolice.org/directives/data/a7a57b38-147b1793-bcd14-7b19-60f8232acf57b5f8.html">CPD</a>.</p>
+    <p>It is unclear where exactly <a href="http://directives.chicagopolice.org/directives/data/a7a57b38-147b1793-bcd14-7b19-60f8232acf57b5f8.html">CPD</a> has applied social media monitoring. However, CPD attempts to use social media monitoring to disrupt <a href="https://www.nbcchicago.com/news/local/aclu-raises-questions-about-chicago-police-social-media-surveillance/2749/">gang and gun violence</a>, so the practice likely disproportionately targets communities of color.</p>
```

Comment:
This is a small nit but I don't think we should state that CPD is using SOCMINT to disrupt gang violence. They _claim_ that as the purpose of social media monitoring (and therefore why its legitimate) but we know they also target protests and other activities.